### PR TITLE
docs: add v2020.1.2 release notes

### PR DESCRIPTION
**Pull request by mweinelt**
Here are the preliminary release notes for Gluon v2020.1.2

Relates to #1981

### CODE REVIEW

File: docs/releases/v2020.1.2.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  The current API is inconsistent and will be replaced eventually. The old API will still be supported for a while.
+
+- Frequent reboots due to out-of-memory or high load due to memory pressure on weak hardware especially in larger
+  meshes (`#1243 <https://github.com/freifunk-gluon/gluon/issues/1243>`_)
+
+  Optimizations in Gluon 2018.1 have significantly improved memory usage.
+  There are still known bugs leading to unreasonably high load that we hope to
+  solve in future releases.
+
+- High chance of ending in a soft-bricked state for Ubiquiti EdgeRouter-X. Workaround is to
```
Comment (blocktrron) [inline docs/releases/v2020.1.2.rst:75]: This is a duplicate

File: docs/releases/v2020.1.2.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Removed hardware support
+------------------------
+
+lantiq-xway
+~~~~~~~~~~~
+
+- AVM FRITZ!Box 7320
+- AVM FRITZ!Box 7330
+- AVM FRITZ!Box 7330 SL
```
Comment (blocktrron) [inline docs/releases/v2020.1.2.rst:12]: Linking the issue could be helpful. There was a proposed patch which nobody took interest in.

File: docs/releases/v2020.1.2.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+.. [#switchports_not_working]
+  The switchports on these devices are not working properly (`#1943 <https://github.com/freifunk-gluon/gluon/issues/1943>`_)
+
+Bugfixes
+--------
+
+- Fixes a bug in the tunneldigger watchdog where the watchdog would incorrectly find itself while looking up the running tunneldigger process. It then went on and assumed a PID mismatch between the tunneldigger service and its PID file and therefore caused an unnecessary restart of the tunnel. (`#1952 <https://github.com/freifunk-gluon/gluon/issues/1952>`_)
+
+- Fixes an oversight in the firewalling of the respondd service where queries from the ``extra_prefix6`` would be dropped. (`#1941 <https://github.com/freifunk-gluon/gluon/issues/1941>`_)
```
Comment (rotanid) [inline docs/releases/v2020.1.2.rst:22]: extra_prefixes6

File: docs/releases/v2020.1.2.rst [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+- Fixes a bug in ``gluon-web`` where forms would not correctly update their field visibility on reset. This affected for example the private wifi page in the config mode. (`#1970 <https://github.com/freifunk-gluon/gluon/pull/1970>`_)
```
Comment (rotanid) [inline docs/releases/v2020.1.2.rst:24]: for example enclosed with commas, or at the end of the sentence with one comma

File: docs/releases/v2020.1.2.rst (hunk 3 above)
Comment (rotanid) [reply docs/releases/v2020.1.2.rst:22]: by the way, why isn't this mentioned in the site.rst doc...

Comment (mweinelt) [reply docs/releases/v2020.1.2.rst]: > extra_prefixes6

Done.

> by the way, why isn't this mentioned in the site.rst doc...

PR welcome.

**Comment by belzebub40k** [comment]
Maybe add  #1977 to the know issues section with a note that _urngd_ can be removed by adding `-urngd` to `GLUON_SITE_PACKAGES` until Gluon switches to _urngd_.

**Comment by mweinelt** [comment]
We'd need to backport that as well, as we would wan't urgnd to be stopped/started properly during autoupdates.

**Comment by belzebub40k** [comment]
> We'd need to backport that as well, as we would want urgnd to be stopped/started properly during autoupdates.

I did not mean adding urngd in favor of haveged. It meant remove urngd in the site.mk to prevent two random number generator from running on the node. Sorry if this was not clear enough.

```
GLUON_SITE_PACKAGES := \
	...
	haveged \
	-urngd
```

**Comment by blocktrron** [comment]
I think this should be done in v2020.2.

**Comment by mweinelt** [comment]
@belzebub40k FWIW: You can always pick cherry-pick those three commits into your v2020.1.x build and migrate to urngd asap if you want it.

```

commit 6a371d88f0a05af7b470afd1f84f650c86a18d51
Author: Jan Luebbe <sho@stratum0.net>
Date:   Wed Apr 22 20:29:10 2020 +0200

    gluon-setup-mode: start urngd instead of haveged
    
    OpenWRT 19.07 enables urngd by default, so haveged is redundant.

commit 35b4a97ccecbc5818f29606cbdd93a0cd7ca7839
Author: Jan Luebbe <sho@stratum0.net>
Date:   Wed Apr 22 20:28:21 2020 +0200

    gluon-autoupdater: start/stop urngd instead of haveged
    
    OpenWRT 19.07 enables urngd by default, so haveged is redundant.

commit 2829c852efd399b699f5b175957f6e6159c53163
Author: Jan Luebbe <sho@stratum0.net>
Date:   Wed Apr 8 10:37:57 2020 +0200

    docs: examples: remove haveged from GLUON_SITE_PACKAGES
    
    OpenWRT 19.07 enables urngd by default, so haveged is redundant.
```

**Comment by NeoRaider** [comment]
Maybe we should add `-urngd` to the `targets/generic` package list for v2020.1.x? I guess it could be considered a bug/regression that we have two rng daemons...

**Comment by rotanid** [comment]
maybe integrate #2002 in the release notes and then do the v2020.1.2 release this weekend?

**Comment by mweinelt** [comment]
> Maybe we should add `-urngd` to the `targets/generic` package list for v2020.1.x? I guess it could be considered a bug/regression that we have two rng daemons...

Merged #2002.

> maybe integrate #2002 in the release notes and then do the v2020.1.2 release this weekend?

Done.

**Pull request merged**